NEWS

```diff
@@ -1,3 +1,26 @@
+0.25.0 (stable)
+ * add PKCS#11 3.0 support [PR#458, PR#461, PR#462, PR#463, PR#464, PR#467, PR#469, PR#470, PR#475, PR#486]
+ * add support for profile objects [PR#479]
+ * add ability to adjust module and config paths at run-time via system environmental exports [PR#442]
+ * make terminal output nicer [PR#509, PR#510]
+ * p11-kit: add command to print merged configuration [PR#446, PR#489]
+ * p11-kit: add commands to list, add and delete profiles of a token [PR#500, PR#503, PR#506]
+ * trust: add command to check format of .p11-kit files [PR#476, PR#483]
+ * trust: lower Cryptoki version from 3.0 to 2.40 [PR#485]
```

Comment:
I think this does not need to be mentioned as it's a follow-up fix after PKCS#11 3.0 support